### PR TITLE
DAQ CRV update

### PR DESCRIPTION
**Pull request by ehrlich-uva**
It has a temporary solution in ArtFragmentsFromDTCEvents_module that deals with the problem, that the DTC_SubEventHeader uses `DTCLib::DTC_Subsystem::DTC_Subsystem_Tracker` instead of `DTCLib::DTC_Subsystem::DTC_Subsystem_CRV` for the Subsystem ID of CRV data.

### CODE REVIEW

**Comment by kutschke** [comment]
@FNALbuild run build test

**Comment by ehrlich-uva** [comment]
We use these modules to convert artFragments (from binary files) to digis. So they are not part of the simulation chain - unless we convert simulated digis into binaries and back to digis. Giani (ArtFragmentsFromDTCEvents_module is his module), Eric and Simon could review the code.

File: DAQ/src/CrvDigisFromFragments_module.cc [hunk 1]
```diff
@@ -103,38 +103,41 @@ void CrvDigisFromFragments::produce(Event& event) {
...
       auto block = crvFragment.dataAtBlockIndex(iDataBlock);
-      if (block == nullptr) {
+      if(block == nullptr)
+      {
         std::cerr << "Unable to retrieve block " << iDataBlock << "!" << std::endl;
         continue;
       }
       auto header = block->GetHeader();
-      if (header->GetSubsystemID() != 2) {
+      if(header->GetSubsystemID() != 2)
```
[the remark this answers is not in the document]
Comment (ehrlich-uva) [reply DAQ/src/CrvDigisFromFragments_module.cc:120]: See above comment.

**Comment by ehrlich-uva** [comment]
@kutschke The two significant changes are 
(1) The temporary lines of code in ArtFragmentsFromDTCEvents_module that I needed to include until the DTC header can be fixed.
(2) I added a setup_event() at https://github.com/Mu2e/Offline/blob/da41a119893ba1cd44100b8ed606f7fd9b21be8c/DAQ/src/CrvDigisFromFragments_module.cc#L109

@eflumerf Other modules (calo, tracker) don't seem to require the setup_event() as far as I can tell. When I run it without the setup_event(), the header has wrong content in about half the times I run the code and works fine at all other times.

**Comment by ehrlich-uva** [comment]
I made the changes and retested the code.

**Comment by kutschke** [comment]
@FNALbuild run build test

**Pull request merged**